src/m_menu.c

```diff
@@ -3488,18 +3498,24 @@ static setup_menu_t* enem_settings[] =
 };
 
 enum {
+  enem1_helpers,
+  enem1_gap1,
+
   enem1_title1,
   enem1_colored_blood,
   enem1_flipcorpses,
   enem1_ghost,
   enem1_fuzz,
-
-  enem1_end
 };
 
 setup_menu_t enem_settings1[] =  // Enemy Settings screen
 {
-  {"Cosmetic", S_SKIP|S_TITLE, m_null, M_X, M_Y},
+  {"Player helpers", S_MBF|S_THERMO|S_THRM_SIZE4|S_LEVWARN|S_ACTION,
```

Comment:
Is "Player Helpers" descriptive enough? If not, some random ideas:
```
Helper Bots
AI Companions
AI Followers
AI Helpers
AI Friends
NPC Companions
```

Reply:
I like `AI Friends`, but perhaps we shouldn't use the term `AI` these days 😆

Reply:
Let's please avoid using buzz words in our menu. :wink: Doesn't MBF explicitly call them "helper dogs"?

Reply:
"Number Of Single-Player Helper Dogs" 😮

Reply:
So, just "Helper Dogs"?

Reply:
That'd be my choice.

Reply:
Play a barking sound whenever the value changes. 😁

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1425/commits/5e9636cdbf50200f1344466e300208b0a3b59b52

Reply:
> Play a barking sound whenever the value changes. 😁

Done in https://github.com/fabiangreffrath/woof/pull/1425/commits/13f68772ab35596557d5fe024354da1935383c87